### ISSUE TITLE
Night mode contrast of blue text not readable.

### ISSUE THREAD

**Issue by David-Rodriguez-Garcia** [mdn/bob#774]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate

### What information was incorrect, unhelpful, or incomplete?

Night mode contrast of blue text not readable. The rotate kw for example.

### What did you expect to see?

Expected to read it properly.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/transform-function/rotate`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/transform-function/rotate/index.md
* Last commit: https://github.com/mdn/content/commit/345cf4c7023bae3a64904f6a27fc5171a6d2a6ae
* Document last modified: 2022-04-19T21:12:12.000Z

</details>

**Comment by wbamberg** [comment]
Thanks for filing!

This is a platform issue, so I'm transferring it to the mdn/yari repository.

**Comment by caugner** [comment]
## Screenshot:

<img width="781" alt="image" src="https://user-images.githubusercontent.com/495429/166435277-b086a6ac-af9b-4e12-928f-fe894d0e3a90.png">

This is an issue within mdn/bob, which can probably be fixed by using CSS variables here:

https://github.com/mdn/bob/blob/b0b44fd5fa362ca717eac2318c2f5674cd6a7d04/editor/css/editor-libs/prism-override.css#L40-L57

**Comment by caugner** [comment]
Possible variables to use:

## `var(--accent-primary)`

<img width="775" alt="image" src="https://user-images.githubusercontent.com/495429/166439067-b62f6c08-7d1f-429c-b8be-6abc87ded90b.png">

<img width="775" alt="image" src="https://user-images.githubusercontent.com/495429/166439078-1e9f010d-8a72-49c8-b4d1-531e04d2fd4a.png">

## `var(--code-token-attribute-name)`

(To be consistent with code blocks)

<img width="775" alt="image" src="https://user-images.githubusercontent.com/495429/166439339-cfd79ea1-1c31-48a1-ae17-59bbf965ab3a.png">

<img width="775" alt="image" src="https://user-images.githubusercontent.com/495429/166439321-41ee6a36-f121-43ee-ab54-d92dac94eb6f.png">

On the other hand, it's already not ideal that properties and functions have the same color:

<img width="775" alt="image" src="https://user-images.githubusercontent.com/495429/166439566-d691e685-4c2b-4a6c-8370-d74524a5061c.png">

@schalkneethling Any opinion? (In doubt, I would go with `var(--accent-primary)` to keep the color similar.)

**Comment by schalkneethling** [comment]
@caugner The best I could find as part of our current colour palette is:

```
$mdn-color-light-theme-blue-40: #5e9eff; for dark mode
```

and 

```
$mdn-color-light-theme-blue-60: #0069c2; for light mode
```

It looks like `accent-primary` is mapped to `$mdn-color-light-theme-blue-50: #0085f2;` which fails even AA contrast in both light and dark mode.

**Comment by NiedziolkaMichal** [comment]
I think --code-token-attribute-name would fit the best, it is already used on functions in Syntax:
![image](https://user-images.githubusercontent.com/100634371/166915850-ab3a9774-9024-4b8b-be17-108225957e26.png)

**Comment by caugner** [comment]
@schalkneethling Would you prefer to stick with blue, or to be consistent with other non-interactive code blocks?

**Comment by schalkneethling** [comment]
> @schalkneethling Would you prefer to stick with blue, or to be consistent with other non-interactive code blocks?

We should stay consistent with regard to code examples as far as possible. Especially with the fragmentation, we have at the moment across repositories.

**Comment by caugner** [comment]
> We should stay consistent

@schalkneethling To clarify: Consistent (a) within _interactive_ examples, or (b) within examples on a page?

(a) would mean implementing [what you suggested](https://github.com/mdn/bob/issues/774#issuecomment-1116025641).
(b) would mean using `--code-token-attribute-name` (well, or something similar, which depends on the accent color of the MDN section we're on).

**Comment by schalkneethling** [comment]
Ideally, we want to use the same color palette across all code examples, irrespective of whether it is interactive or not. The main criteria for me are that the colors meet at least AA WCAG compliance for color contrast. It might mean that we need to change some colors both here and in Yari.

I am happy to help identify colors that would work well.

**Comment by caugner** [comment]
> the colors meet at least AA WCAG compliance for color contrast

Do the browser developer tools tell me this, or how can I check this? 🙂

**Comment by schalkneethling** [comment]
> > the colors meet at least AA WCAG compliance for color contrast
> 
> Do the browser developer tools tell me this, or how can I check this? 🙂

Chrome’s tools are good at this, as is Firefox, but I use this tool, https://www.tpgi.com/color-contrast-checker/
Here is also a nice selection of tools https://developer-toolchest.com/?q=a11y
